web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -112,8 +118,20 @@ export const SidebarCardsList = (props: Props): ReactElement => {
 
   return (
     <>
-      <h2 className="h1-styling margin-top-0">{Config.dashboardDefaults.sidebarHeading}</h2>
-      <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+      {isDesktopAndUp ? (
```

Comment:
Question about this implementation. Further up in `dashboard.tsx` we do a check for `isDesktopAndUp` and then render the dashboard either for mobile or desktop. Both contain independent implementations of `SidebarCardsContainer` which contains `SidebarCardsList`. I feel like I see the current approach pretty commonly throughout our application, where we just do the check for what we need in the component. In this case though, I wonder if it makes sense to pass a boolean down from `dashboard.tsx` where we could set this when invoking the parent.

Is there a reason we typically follow this approach? Obviously fewer props is easier to read and this isn't an intensive operation. @cbrunefearless @aloverso

Reply:
How do you feel about saving this as a refactor?

Reply:
Either way. I'm not too partial. Mainly curious if there was a preference on the team.